torch/nn/modules/module.py

```diff
@@ -1826,6 +1828,13 @@ def _call_impl(self, *args, **kwargs):
 
             return result
 
+        from torch.compiler import is_compiling
+
+        if is_compiling():
```

Comment:
Can we have a comment here that explains why this is ok (what is explained in the PR description)?